.travis.yml

```diff
@@ -5,14 +5,15 @@ android:
     - platform-tools                    # Platform tools
     - build-tools-28.0.3                # Build tools version
     - android-28                        # Target SDK version
-    - sys-img-armeabi-v7a-android-18    # Emulator
+    - sys-img-x86-android-28            # Emulators
+    - sys-img-x86-android-23
+    - sys-img-x86-android-21
```

Comment:
looks like we also need to manually start the emulators

 - [ ] start emulators